Update with style guide (#1133)
iotaledger/hornet: docs/post_installation/managing.md

  },
```

> You can also control plugins can be controlled using the [Dashboard/web interface](./post_installation.md#Dashboard).

Reviewer: ```suggestion
You can also control plugins using the [Dashboard/web interface](./post_installation.md#Dashboard).
```

## Spammer

> Hornet integrates a lightweight spamming plugin that spams the network with messages. The IOTA network is based on Directed Acyclic Graph.  Therefore, new incoming messages are connected to previous messages (tips). It is healthy for the network to maintain some level of message rate.

Reviewer: ```suggestion
Hornet integrates a lightweight spamming plugin that spams the network with messages. The IOTA network is based on Directed Acyclic Graph. Therefore, new incoming messages are connected to previous messages (tips). It is healthy for the network to maintain some level of message rate.
```

  },
```

### Snapshot Pruning
During a snapshot, Hornet may delete messages from the ledger if they were confirmed by an old milestone. In other words, the term _pruning_ means the deletion of the old history from the node database.  

* If you want to enable pruning, you should set the `pruning.milestones.enabled` or `pruning.size.enabled` keys to _enabled_.
* The `pruning.milestones.maxMilestonesToKeep` defines how far back from the current confirmed milestone should be pruned.
* The `pruning.size.targetSize` defines the maximum database size.  Old data will be pruned.

There are two types of snapshots:

#### Delta snapshots

> A delta snapshot points to a specific full snapshot, this means a delta snapshot consists of the changes since the last full snapshot.

Reviewer: ```suggestion
A delta snapshot points to a specific full snapshot, meaning a delta snapshot consists of the changes since the last full snapshot.
```
